pom.xml

```diff
@@ -17,7 +17,7 @@
   <name>mango: an ADAM-based genomics visualization toolkit</name>
 
   <properties>
-    <adam.version>0.17.1-SNAPSHOT</adam.version>
+    <adam.version>0.17.1</adam.version>
```

Comment:
Oops! Sorry about this. Our CI caught this (the snapshot releases cease to exist once a real release is made) but I had the emailer set up incorrectly. I've fixed that so that this won't be a problem moving forward.